examples/AnnotatorForClinicalDataExample.java

```diff
@@ -0,0 +1,543 @@
+
+		Flow flow = new Flow.Builder()
+			.elements(Arrays.asList(cdFlowEntry, negFlowEntry))
+			.async(true)
+			.build();
+
+		AnnotatorFlow annotatorFlow = new AnnotatorFlow.Builder()
+			.flow(flow)
+			.build();
+
+		AcdFlow acdFlow = new AcdFlow.Builder()
+				.id("my_flow")
+				.name("My flow name")
+				.description("My flow desc")
+				.annotatorFlows(Arrays.asList(annotatorFlow))
+				.build();
+
+		CreateFlowsOptions options = new CreateFlowsOptions
+			.Builder()
+			.acdFlow(acdFlow)
+			.build();
+
+		ServiceCall<Void> call = acd.createFlows(options);
+		Response<Void> resp = call.execute();
+		System.out.println("Response code: " + resp.getStatusCode()); // should be 201
+
+	}
+
+	// PUT /flows
+	public static void updateFlow() {
+
+		Annotator conceptDetection = new Annotator.Builder()
+			.name(Annotator.Name.CONCEPT_DETECTION).build();
+
+		Annotator hypothetical = new Annotator.Builder()
+			.name(Annotator.Name.HYPOTHETICAL).build();
+
+		FlowEntry cdFlowEntry = new FlowEntry
+			.Builder().annotator(conceptDetection).build();
+
+		FlowEntry hypoFlowEntry = new FlowEntry
+			.Builder().annotator(hypothetical).build();
+
+		Flow flow = new Flow.Builder()
+			.elements(Arrays.asList(cdFlowEntry, hypoFlowEntry))
+			.async(true)
+			.build();
+
+		AnnotatorFlow annotatorFlow = new AnnotatorFlow.Builder()
+			.flow(flow)
+			.build();
+
+		AcdFlow acdFlow = new AcdFlow.Builder()
+			.id("my_flow")
+			.name("My flow name - updated")
+			.description("My flow desc - updated")
+			.annotatorFlows(Arrays.asList(annotatorFlow))
+			.build();
+
+		UpdateFlowsOptions opts = new UpdateFlowsOptions.Builder()
+			.id("my_flow")
+			.acdFlow(acdFlow)
+			.build();
+
+		ServiceCall<Void> call = acd.updateFlows(opts);
+		Response<Void> resp = call.execute();
+		System.out.println("Response code: " + resp.getStatusCode()); // should be 200
+
+
+	}
+
+	// DELETE /flows/{id}
+	public static void deleteFlow() {
+
+		DeleteFlowsOptions opts = new DeleteFlowsOptions
+			.Builder()
+			.id("my_flow")
+			.build();
+
+		ServiceCall<Void> call = acd.deleteFlows(opts);
+		Response<Void> resp = call.execute();
+		System.out.println("Response code: " + resp.getStatusCode());
+
+	}
+
+	// DELETE /user_data
+	public static void deleteTenantData() {
+
+		DeleteUserSpecificArtifactsOptions opts = new DeleteUserSpecificArtifactsOptions();
+		ServiceCall<Void> call = acd.deleteUserSpecificArtifacts(opts);
+		Response<Void> resp = call.execute();
+		System.out.println("Response code: " + resp.getStatusCode());
+
+	}
+
+	// GET /status/health_check
+	public static void getHealthCheck() {
+
+		ServiceCall<ServiceStatus> call = acd.getHealthCheckStatus();
+		Response<ServiceStatus> resp = call.execute();
+		System.out.println("Response code: " + resp.getStatusCode() + " Status: " + resp.getStatusMessage());
+
+	}
+
+	// GET /cartridges
+	public static void getCartridges() {
+
+		CartridgesGetOptions opts = new CartridgesGetOptions();
+		/*
```

Comment:
Why is this the code for this method commented out?

Reply:
The List<AcdCartridges> response from cartridgesGet(CartridgesGetOptions) wasn't merged into master at the time. The code has now been merged into master, but it has yet to be published in mvn central. So, the code will compile if you clone master locally, but not if you're consuming <= v1.1.0 of the sdk.